config/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/SIGBlockchain/project_aurum/internal/config"
+)
+
+func main() {
+	//open configuration file
+	configFile, err := os.Open("../cmd/config.json")
+	if err != nil {
+		log.Fatal("Failed to load configuration file : " + err.Error())
+	}
+	defer configFile.Close()
+
+	cfgData, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		log.Fatal("Failed to read configuration file : " + err.Error())
+	}
+
+	cfg := config.Config{}
+	if err := json.Unmarshal(cfgData, &cfg); err != nil {
+		log.Fatal("Failed to unmarshall configuration data : " + err.Error())
+	}
+
+	//specify flags
+	versionU64 := flag.Uint64("version", uint64(cfg.Version), "enter version number")
+	flag.Uint64Var(&cfg.InitialAurumSupply, "initialaurumsupply", cfg.InitialAurumSupply, "enter a number for initial aurum supply")
+	flag.StringVar(&cfg.Port, "port", cfg.Port, "enter port number")
+	flag.StringVar(&cfg.BlockProductionInterval, "blockproductioninterval", cfg.BlockProductionInterval, "enter a time for block production interval")
+	flag.BoolVar(&cfg.Localhost, "localhost", cfg.Localhost, "enter either true/false")
```

Comment:
The `localhost` flag doesn't appear to be working. I tried `./config -localhost false` and `./config -localhost 0` and it didn't change.

Reply:
Oh that's because the syntax `-flag x` is for non-boolean flags only. The flag works if you use the following syntax: `-flag=x`. I can change it to accept the other syntax too if you like.

Reply:
I think keeping it simple is best, but make sure to include the correct syntax in the help string.